lib-multisrc/colamanga/src/eu/kanade/tachiyomi/multisrc/colamanga/ColaManga.kt

```diff
@@ -47,25 +63,31 @@ abstract class ColaManga(
 
     override val supportsLatest = true
 
-    private val json: Json by injectLazy()
-
-    private val intl = ColaMangaIntl(lang)
-
+    private var pages: MutableMap<String, ArrayList<Page>> = mutableMapOf()
+    private val intl = Intl(
+        Locale.getDefault().language,
+        setOf("en", "zh"),
+        lang,
+        javaClass.classLoader!!,
+    )
     private val preferences by getPreferencesLazy()
-
-    override val client = network.cloudflareClient.newBuilder()
-        .rateLimitHost(
-            baseUrl.toHttpUrl(),
-            preferences.getString(RATE_LIMIT_PREF_KEY, RATE_LIMIT_PREF_DEFAULT)!!.toInt(),
-            preferences.getString(RATE_LIMIT_PERIOD_PREF_KEY, RATE_LIMIT_PERIOD_PREF_DEFAULT)!!.toLong(),
-            TimeUnit.MILLISECONDS,
-        )
-        .addInterceptor(ColaMangaImageInterceptor())
-        .build()
-
-    override fun headersBuilder() = super.headersBuilder()
-        .add("Origin", baseUrl)
-        .add("Referer", "$baseUrl/")
+    var webView: MutableMap<String, WebView> = mutableMapOf()
+    val interfaceName = randomString()
+
+    override val client = network.cloudflareClient.newBuilder().rateLimitHost(
+        baseUrl.toHttpUrl(),
+        preferences.getString(RATE_LIMIT_PREF_KEY, RATE_LIMIT_PREF_DEFAULT)!!.toInt(),
+        preferences.getString(RATE_LIMIT_PERIOD_PREF_KEY, RATE_LIMIT_PERIOD_PREF_DEFAULT)!!
+            .toLong(),
+        TimeUnit.MILLISECONDS,
+    ).setRandomUserAgent(
+        preferences.getPrefUAType(),
+        preferences.getPrefCustomUA(),
+        filterInclude = listOf("Chrome"),
+    ).addInterceptor(DataImageInterceptor()).build()
+
+    override fun headersBuilder() =
+        super.headersBuilder().add("Origin", baseUrl).add("Referer", "$baseUrl/")
```

Comment:
Refrain from introducing alternative formatting unless the original code broke any of the linting rules. This specific PR is a big code change and formatting changes are very noisy.

Please revert any unnecessary formatting changes, including the changes below.